espresso/submitter/multi_worker_queue_espresso_submitter.go

```diff
@@ -0,0 +1,1175 @@
+package submitter
```

Comment:
This file has become overall pretty big, I wonder if we can somehow split the code?